PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/ui/SurveyPopUpDialog.java

```diff
@@ -0,0 +1,224 @@
+package com.microsoft.azuretools.ijidea.ui;
+
+import com.intellij.ide.ui.LafManager;
+import com.intellij.ide.ui.LafManagerListener;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.project.impl.ProjectManagerImpl;
+import com.intellij.openapi.util.IconLoader;
+import com.intellij.openapi.wm.impl.WindowManagerImpl;
+import com.intellij.ui.HyperlinkLabel;
+import com.microsoft.intellij.helpers.CustomerSurveyHelper;
+import com.microsoft.intellij.util.PluginUtil;
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.*;
+import javax.swing.event.MouseInputAdapter;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseMotionAdapter;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+
+public class SurveyPopUpDialog extends JDialog {
```

Comment:
I prefer to add telemetry in `CustomerSurveyHelper` if telemetry is required, for `AzureDialogWrapper` mainly collect users' input like the value of `TextFiled`, `RadioButton` and so on, which didn't exist in pop up window, and we need to override the `getActions` in case of extra OK/Cancel buttons.